browser/ui/android/strings/translations/android_brave_strings_en-GB.xtb

```diff
@@ -122,7 +122,7 @@
 <translation id="5441565662498481293">You are automatically sending a tip to:</translation>
 <translation id="1166682575529520472">Monthly</translation>
 <translation id="1628742278973105368">Your first monthly tip will be sent on:</translation>
-<translation id="298349922412901215">NOTE: this creator has not yet signed up to receive contributions from Brave users. We'll keep trying to contribute until they verify, or until 90 days have passed.</translation>
+<translation id="298349922412901215">This creator has not yet signed up to receive contributions from Brave users. We'll keep trying to contribute until they verify, or until 90 days have passed.</translation>
```

Comment:
why do we change strings in translations, aren't they get replaced when we pull new files from transifex? cc @mkarolin

Reply:
This is any how not an issue, currently this text is updated in default.
after replace this will also get correct value.

Reply:
I still don't get why do you modify en-GB translation? That text will be translated and auto updated with other languages translation.

Reply:
removed this change